woocommerce/woocommerce-android
# Change navigation from review details from popback to navigating back to review list

Closes: #9026 


### Description


If the app is opened on another screen other than Reviews, and a new review notification is tapped, there will be no undo snackbar, because the app will navigate to Menu instead of Reviews. This PR changes this behaviour so the app navigates back to the review list screen, and the snackbar is displayed. 

### Testing instructions



1. Make sure notifications are allowed for the app
2. Open any other screen than Reviews, like My Store.
3. On another device/web submit a product review
4. Wait for the notification to appear, and tap the notification
5. In the opened review, tap Trash,  Approve or Spam. 
6. Make sure it navigates back to the Reviews List and the snackbar with the option to UNDO the action is displayed. 

### Images/gif


### Before 

https://github.com/woocommerce/woocommerce-android/assets/30724184/0a8abaaf-b1b8-4170-8e46-78db68a1fa62

### After

https://github.com/woocommerce/woocommerce-android/assets/30724184/1e9c8e71-e3f5-48e4-a3e9-d2cc88dd8c5e
